pkg/minikube/machine/fix.go

```diff
@@ -94,6 +99,22 @@ func fixHost(api libmachine.API, cc *config.ClusterConfig, n *config.Node) (*hos
 	return h, nil
 }
 
+// warnAboutMisconfiguredMachineConfigPaths will warn user if paths in machine config.json are misconfigured.
+// The problem may be caused if config files will be moved manually to other location https://github.com/kubernetes/minikube/issues/14466
+func warnAboutMisconfiguredMachineConfigPaths(h *host.Host) {
+	var driver map[string]interface{}
+	err := json.Unmarshal(h.RawDriver, &driver)
+	if err != nil {
+		return
+	}
+
+	if h.HostOptions.AuthOptions.StorePath != localpath.MiniPath() || driver["StorePath"] != localpath.MiniPath() {
+		out.WarningT(`Your machine config has probably misconfigured some paths. To ensure your paths are updated:
+
+	'minikube update-machine-config'`)
```

Comment:
`-p <profile_name>` should be included in the command

Reply:
Done.